torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6098,6 +6113,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_inplace_autograd=True,
            safe_casts_outputs=True,
            sample_inputs_func=sample_inputs_xlogy),
+    OpInfo('zero_',
```

Comment:
Does this work? AFAIK, there is no operator named `zero_` in torch namespace.
You might need to need wrap the operator in lambda.
https://github.com/pytorch/pytorch/blob/d88d321ee3a65894baf6dc0a580b44180ec52f3c/torch/testing/_internal/common_methods_invocations.py#L4383